modules/medianetAnalyticsAdapter.md

```diff
@@ -0,0 +1,23 @@
+# Overview
+Module Name: media.net Analytics Adapter
+
+Module Type: Analytics Adapter
+
+Maintainer: prebid-support@media.net
+
+# Description
+
+Analytics adapter for Media.net
+https://media.net/
+
+# Test Parameters
+
+```
+{
+  provider: 'medianet',
```

Comment:
provider seems to be incorrect here as analytics code used in adapter file is `medianetAnalytics`

Reply:
@sumit116 have fixed that, please have a look